packages/@o3r/core/schematics/rule-factories/otter-environment/index.ts

```diff
@@ -93,9 +91,8 @@ export function updateOtterEnvironmentAdapter(
       });
 
     }
-    if (options.isDefaultGenerator) {
-      registerCollectionSchematics(workspace, '@o3r/core');
-    }
+    registerCollectionSchematics(workspace, '@schematics/angular');
+    registerCollectionSchematics(workspace, '@o3r/core');
```

Comment:
Should not be the `o3r/core` first ? and then the schematics/angular ?

Reply:
In fact no need to add `@schematics/angular` it's already done in `registerCollectionSchematics`
and `registerCollectionSchematics` is adding the collection on top of the previous registered

Reply:
In fact no need to add @schematics/angular it's already done in registerCollectionSchematics
and registerCollectionSchematics is adding the collection on top of the previous registered